UnityProjects/MRTKDevTemplate/Packages/manifest.json

```diff
@@ -14,13 +14,14 @@
     "com.microsoft.mrtk.input": "file:../../../com.microsoft.mrtk.input",
     "com.microsoft.mrtk.spatialmanipulation": "file:../../../com.microsoft.mrtk.spatialmanipulation",
     "com.microsoft.mrtk.standardassets": "file:../../../com.microsoft.mrtk.standardassets",
+    "com.microsoft.mrtk.tools": "file:../../../com.microsoft.mrtk.tools",
```

Comment:
Do we need to include the tools package in the template project?

Reply:
we do not. it was there for testing

Reply:
It wouldn't hurt to keep it around imo, so we can use the tools ourselves if want!

Reply:
works for me. i will leave it there :)